packages/umu-launcher-git/umu-launcher-git.pacscript

```diff
@@ -0,0 +1,50 @@
+pkgname="umu-launcher-git"
+gives="umu-launcher"
+repology=("project: ${gives}")
+arch=('amd64')
```

Comment:
```suggestion
arch=('any')
```
Sorry I did the original review really early and I should've been more clear: use `any` here, because that will indicate that the package can be built on any architecture but only usable on the architecture that it was built on.